book/security.rst

```diff
@@ -14,11 +14,11 @@ is both flexible and (hopefully) fun to work with.
 Since there's a lot to talk about, this chapter is organized into a few big
 sections:
 
-1) Initial ``security.yml`` setup (*authentication*);
+1. Initial ``security.yml`` setup (*authentication*);
 
-2) Denying access to your app (*authorization*);
+2. Denying access to your app (*authorization*);
 
-3) Fetching the current User object
+3. Fetching the current User object.
```

Comment:
Thanks, will keep it in mind :)
I thought the same, but decide that I don't feel like experimenting so will leave it like it is.
Should I change the numbers to `#` or leave it like it is?